jerry-core/ecma/builtin-objects/ecma-builtin-helpers.c

```diff
@@ -677,13 +677,16 @@ ecma_builtin_helper_def_prop (ecma_object_t *obj_p, /**< object */
   prop_desc.value = value;
 
   prop_desc.is_writable_defined = true;
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wconversion"
   prop_desc.is_writable = writable;
 
   prop_desc.is_enumerable_defined = true;
   prop_desc.is_enumerable = enumerable;
 
   prop_desc.is_configurable_defined = true;
   prop_desc.is_configurable = configurable;
+#pragma GCC diagnostic pop
```

Comment:
It seems this is a bool to undisned int:1 conversion. Do some casts help it?

    prop_desc.is_writable = (unsinged int) writable;
    prop_desc.is_writable = writable ? 1 : 0;